### PR TITLE
[KOGITO-2912] Reorganized scripts

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-2912
https://issues.redhat.com/browse/KOGITO-2855

Added support of different artifacts version (in case runtimes is released but not the images)
Reorganized the scripts for more clarity on which script do what:

- manage-kogito-version.py => Update version / examples_ref / artifacts version
- update-maven-artifacts.py => Only update the service apps artifacts reference
- update-tests.py => update clone-repo.sh and behave tests only with given information

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Review by Kaitou786 — APPROVED** [review]

**Comment by radtriste** [comment]
@spolti can we merge that one ?

**Review by Kevin-Mok — APPROVED** [review]

File: scripts/README.md [hunk 1]
```diff
@@ -96,61 +100,88 @@ This script is called as the last step of the `make push-staging` command define
...
 
- - kogito-data-index/module.yaml
- - kogito-jobs-service/module.yaml
- - kogito-management-console/module.yaml
+This will set the default artifacts version to 1.0.0 into the behave tests.
+
+##### Update Examples URI and Ref
+
+```bash
+$ python update-tests.py --examples-uri https://github.com/<yournamespace>/kogito-examples --examples_uri 1.0.0
```
Comment (Kevin-Mok) [inline scripts/README.md]:
```suggestion
$ python update-tests.py --examples-uri https://github.com/<yournamespace>/kogito-examples --examples-ref 1.0.0
```

**Review by Kevin-Mok — APPROVED** [review]
Only a minor fix.

File: scripts/README.md (hunk 1 above)
Comment (spolti) [reply scripts/README.md]: good catch.

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
@radtriste @Kevin-Mok have found a little issue, we commit his suggestions, will merge once CI is finished.

**Pull request merged**